### PR TITLE
Add prefix to bwa/index

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by nvnieuwk** [comment]
@maxulysse can I merge this? :pleading_face:

**Comment by maxulysse** [comment]
can you do bwamem2/index too?

**Comment by nvnieuwk** [comment]
Done!

File: modules/nf-core/bwa/index/main.nf [hunk 1]
```diff
@@ -11,20 +11,21 @@ process BWA_INDEX {
     tuple val(meta), path(fasta)
 
     output:
-    tuple val(meta), path(bwa) , emit: index
-    path "versions.yml"        , emit: versions
+    tuple val(meta), path("bwa/") , emit: index
+    path "versions.yml"           , emit: versions
```
Comment (maxulysse) [inline modules/nf-core/bwa/index/main.nf:15]: I'd rather we keep that the way it was for more consistency between the different aligners

Comment (nvnieuwk) [reply modules/nf-core/bwa/index/main.nf:15]: fine by me! as long as it works

**Review by maxulysse — APPROVED** [review]

**Pull request merged**